pages/style-guide.md

```diff
@@ -2,11 +2,11 @@
 	"title": "jQuery's Style Guides"
 }</script>
 
-jQuery projects strive to use a common set of styles for code and documentation. This facilitates a team of developers working on the same code base. If each developer were to use their own style, there would be jarring changes in the code as someone read through it. This slows comprehension and can also cause spurious differences in changesets as developers alter style in frivolous ways on the code they touch.
+Style guides aim to make a code base appear as if it were written by one precise punctilious programmer, rather than a cacophony of competing coders. As with many aspects of coding and writing, there can be differences of opinion about the best style. Contentious issues like "spaces versus tabs" or "semicolons or not" distract from the goal of writing code that works.
 
-Our style guides aim to make the code bases appear as if they were written by one precise punctilious programmer, rather than a cacophony of competing coders. As with many aspects of coding and writing, there can be differences of opinion about the best style. Contentious issues like "spaces versus tabs" or "semicolons or not" distract from the goal of writing code that works. Most project contributors have a slightly different personal coding style, but we all use the jQuery style when contributing to jQuery projects.
+Long term, we would like to have formatting automatically done by the build process, so that the resulting code would always follow the style guide. We have taken a few steps in that direction. For example, our projects contain an [EditorConfig file](http://editorconfig.org) that many programming editors can use to enforce the space and tab rules. Our [JSHint](http://www.jshint.com/about/) configuration checks for common style _faux pas_ such as trailing spaces on a line, or using a variable before it is defined. Some projects use [ESLint]() or [JSCS]() to do more exensive style checks.
```

Comment:
> Long term, we would like to have formatting automatically done by the build process

`esformatter`, also used by `standard-format`, can provide that. JSCS with autofixing as well (afaik to a lesser degree though). Since those tools exist and aren't used in our build process (right?), I wonder if it really makes sense to mention that here as a goal.